.gitignore

```diff
@@ -18,4 +18,4 @@ deploy-to-ec2*
 *.ncrunchsolution
 *.orig
 Thumbs.db
-src/Carnac.sln.ide
+src/Carnac.sln.ide/
```

Comment:
Possibly unrelated tangent: this seems to have changed [in VS15 CTP 5]( https://github.com/github/gitignore/pull/1388) to `*.vs/`

What version of VS are you running here?